_episodes/02-filedir.md

```diff
@@ -734,14 +733,18 @@ $ ls -F /
 
 `ls` is the **command**, with an **option** `-F` and an
 **argument** `/`.
-We've already encountered options (also called **switches** or **flags**) which
+We've already encountered options  which
 either start with a single dash (`-`) or two dashes (`--`),
 and they change the behavior of a command.
 [Arguments] tell the command what to operate on (e.g. files and directories).
 Sometimes options and arguments are referred to as **parameters**.
 A command can be called with more than one option and more than one argument, but a
 command doesn't always require an argument or an option.
 
+Note that options are also called **switches** or **flags**, especially for options
```

Comment:
This wording implies that the terms switch and flag are on equal footing with option. I think we're trying to present 'option' as the preferred term to use (because that's what the man pages use), so I would reword to reflect this. Something like
> You might sometimes see  *options* being referred to as **switches** or **flags**, especially for options that take no argument. In this lesson we will stick with using the term *option*.

Reply:
I've no objection to this change being made before merge, but doing so via the pull request resolution interface on github is, at present, beyond my ken.

Reply:
Let me know if you'd like me to make that change, but the normal approach would be for you to make another commit on your feature branch and push it. That will then your the PR.

Reply:
Follow-up commit pushed to branch in forked repo. Was afraid my reliance on command-line tools would force me to learn yet another web interface for a single action.

Reply:
I've squashed and merge this PR now. Thanks again for the contribution!